addons/resource.language.en_gb/resources/strings.po

```diff
@@ -15592,7 +15592,13 @@ msgctxt "#24136"
 msgid "Would you like to enable this add-on?"
 msgstr ""
 
-#empty strings from id 24137 to 24138
+#. Button in Add-on Manager -> Available Updates for updating unpinned add-ons when auto-updating is off.
+#: xbmc/filesystem/AddonDirectory.cpp
+msgctxt "#24137"
+msgid "Update all unpinned add-ons"
```

Comment:
As we now (or will have) after @ronie's PR have a lock symbol on pinned add-ons should we maybe call this `Update all unlocked add-ons`? Or keep it very explicit and call it `Update all auto-updateable add-ons`. Or leave it as is???

Reply:
i vote `Update all auto-updateable add-ons`